### PR TITLE
how use main method

### PR DESCRIPTION
**Pull request by max-java**

### CODE REVIEW

**Comment by KaydunovDenis** [comment]
I don't understand your question.

**Review by KaydunovDenis — CHANGES_REQUESTED** [review]
Не понимаю суть твоего вопроса.

**Comment by max-java** [comment]
> Не понимаю суть твоего вопроса.

Than you'd better be on lecture )

File: lecturer/716_operators/src/main/java/com/tutrit/java/quickstart/Application.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        Logger log = LoggerFactory.getLogger("main");
+        if(args.length > 1) {
+            String command = args[0];
+            switch (command) {
+                case "sum":
+                    if(args.length >= 3) {
+                        OperatorService operatorService = new OperatorService();
+                        operatorService.run(Integer.parseInt(args[1]), Integer.parseInt(args[2]));
+                    }
+                    break;
```
Comment (KaydunovDenis) [inline lecturer/716_operators/src/main/java/com/tutrit/java/quickstart/Application.java:20]: Situation with lenght = 2 don't will you check?

File: lecturer/716_operators/src/test/java/com/tutrit/java/quickstart/ApplicationTest.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.tutrit.java.quickstart;
+
+
+import org.junit.Test;
+
+public class ApplicationTest {
+
+    @Test
+    public void main() {
+    }
```
Comment (KaydunovDenis) [inline lecturer/716_operators/src/test/java/com/tutrit/java/quickstart/ApplicationTest.java:10]: Need realisation. I think it Maks want show on lecture.

File: lecturer/716_operators/src/test/java/com/tutrit/java/quickstart/ApplicationTest.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package com.tutrit.java.quickstart;
+
```
Comment (KaydunovDenis) [inline lecturer/716_operators/src/test/java/com/tutrit/java/quickstart/ApplicationTest.java:2]: unnecessary blank line

File: lecturer/716_operators/src/main/java/com/tutrit/java/quickstart/Application.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                default:
+                    log.info("{} not supported", command);
+            }
+        }
+
+    }
+
+    private void propertiesExample(String[] args) {
```
Comment (KaydunovDenis) [inline lecturer/716_operators/src/main/java/com/tutrit/java/quickstart/Application.java:28]: What is this method?
I don't can find where it using.

**Review by KaydunovDenis — DISMISSED** [review]
Where is the task description?

See my comment.

**Review by VasiliZas — APPROVED** [review]

**Review by KaydunovDenis — APPROVED** [review]

**Review by PavelBoPavel — APPROVED** [review]

**Pull request merged**